templates/python/http/func.py

```diff
@@ -9,5 +18,12 @@ def main(context: Context):
     """
 
     # Add your business logic here
+    print("Received request")
 
-    return { "message": "Howdy!" }, 200
+    if 'request' in context.keys():
+        ret = pretty_print(context.request)
+        print(ret)
+        return { "message": "hello" }, 200
```

Comment:
Hey lance, yes, you're probably right! I changed it back from an earlier commit after seeing the e2e-tests matching on fixed strings like `hello`, `howdy` or derivations of that, so I thought *printing* the response and *returning* a short answer was the way the templates were supposed to behave.

I'll gladly update the template so that it returns the actual response, probably some time tomorrow.